src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -29,6 +29,9 @@ public static partial class PlatformDetection
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
         public static bool IsAndroid => RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"));
         public static bool IsiOS => RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"));
+        public static bool IsAndroidAOT => IsAndroid && IsMonoAOT;
+        public static bool IsNotAndroidAOT => !IsAndroidAOT;
+        public static bool IsNotDeviceAOT => !((IsAndroid || IsiOS) && IsMonoAOT);
```

Comment:
Do we expect this to include tvOS and MacCatalyst ? Is Device specifically Android and iOS (excluding tvOS MacCatalyst)?

Reply:
I'm not sure, no issue to include them here or it can be done in a follow-up @steveisok

Reply:
I think naming-wise, `IsNotDeviceAOT` may suggest the platform is `not AOT compiled iOS, Android, or tvOS`, but in this PR, it's `not AOT compiled iOS or Android`. I'm curious if we intend to skip the same test methods if on AOT tvOS (if we plan to have AOT tvOS).

Reply:
In order to be consistent with naming, I can add `tvOS` too.